### PR TITLE
Terms of Use

### PR DESCRIPTION
**Pull request by Wilfongjt**

### CODE REVIEW

File: notebook/.ipynb_checkpoints/00.terms.of.use.text.2.haml-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.1"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (jacebrowning) [inline notebook/.ipynb_checkpoints/00.terms.of.use.text.2.haml-checkpoint.ipynb]: I don't think this file is meant to be tracked in Git.

Comment (Wilfongjt) [reply notebook/.ipynb_checkpoints/00.terms.of.use.text.2.haml-checkpoint.ipynb]: Could you provide a little more maybe you thoughts as to why?

Comment (jacebrowning) [reply notebook/.ipynb_checkpoints/00.terms.of.use.text.2.haml-checkpoint.ipynb]: Here's the question being asked on Stack Overflow: https://stackoverflow.com/questions/36306017/should-ipynb-checkpoints-be-stored-in-git

It's also part of GitHub default ignore file for Python projects: https://github.com/github/gitignore/blob/master/Python.gitignore

Comment (Wilfongjt) [reply notebook/.ipynb_checkpoints/00.terms.of.use.text.2.haml-checkpoint.ipynb]: Yes, yes, yes! I  didnt know the check points are independent from the notebook. I will give it a go. T

**Pull request merged**